src/app/accounts/platforms.ts

```diff
@@ -105,36 +105,14 @@ function loadPlatforms(membershipId: string): ThunkResult<readonly DestinyAccoun
   };
 }
 
-function loadActivePlatform(): ThunkResult<DestinyAccount | undefined> {
-  return async (_dispatch, getState) => {
-    const account = currentAccountSelector(getState());
-    if (account) {
-      return account;
-    }
-
-    const accounts = accountsSelector(getState());
-    if (!accounts.length) {
-      return undefined;
-    }
-
-    const membershipId = localStorage.getItem('dim-last-membership-id');
-    const destinyVersionStr = localStorage.getItem('dim-last-destiny-version');
-    const destinyVersion = destinyVersionStr ? parseInt(destinyVersionStr, 10) : 2;
-
-    const active = accounts.find(
-      (account) =>
-        account.membershipId === membershipId && account.destinyVersion === destinyVersion
-    );
-
-    return active ?? _.maxBy(accounts, (account) => account.lastPlayed);
```

Comment:
This line was important to remove - it would say "well if we can't find the last account you were on, I guess pick the most recent". Now we always go to the one you picked regardless.

Since we now store the last membership ID and destiny version directly in the Redux store (and load it immediately from localStorage) we don't need this function at all.